Examples/ObjectiveC/ShapeCollectionFeatureExample.m

```diff
@@ -0,0 +1,66 @@
+#import "ShapeCollectionFeatureExample.h"
+@import Mapbox;
+
+NSString *const MBXExampleShapeCollectionFeature = @"ShapeCollectionFeatureExample";
+
+@interface ShapeCollectionFeatureExample () <MGLMapViewDelegate>
+@property (nonatomic) MGLMapView *mapView;
+@end
+
+@implementation ShapeCollectionFeatureExample
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    self.mapView = [[MGLMapView alloc] initWithFrame:self.view.bounds styleURL:[MGLStyle lightStyleURLWithVersion:9]];
```

Comment:
If we use `MGLStyleDefaultVersion` anywhere in our documentation, I’d accompany it with a comment explaining that we know what we’re doing – “don’t use this at home” if you’re using runtime styling.